frontend/sw360-portlet/src/main/webapp/html/utils/includes/usingProjectsTable.jspf

```diff
@@ -9,29 +9,55 @@
 --%>
 
 <core_rt:if test="${usingProjects != null  && usingProjects.size()>0}">
-    <table class="table info_table" title="Usage overview">
+    <table class="table info_table" style="margin-bottom: 0; border-bottom-width: 0">
```

Comment:
Just so the header and the table don't have space between them, otherwise it looks like this:
![screenshot from 2017-07-26 13-15-50](https://user-images.githubusercontent.com/30042647/28618423-a489c390-7204-11e7-8dc2-8a188a7d6e27.png)
The header can't go in the table because datatables requires more than one cell per row.